src/Countries/Moldova.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Moldova extends Country
+{
+    public function countryCode(): string
+    {
+        return 'md';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Anul Nou' => '01-01',
+            'Crăciunul pe stil vechi' => '01-07',
+            'A doua zi de Crăciun pe stil vechi' => '01-08',
+            'Ziua Internațională a Femeii' => '03-08',
+            'Ziua Muncii' => '05-01',
+            'Ziua Europei' => '05-09',
+            'Ziua Internațională a Copilului' => '06-01',
+            'Ziua Independenței' => '08-27',
+            'Ziua Limbii Române' => '08-31',
+            'Crăciunul pe stil nou' => '12-25',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter =  $this->orthodoxEaster($year);
+
+        return [
+            'Prima zi de Paște' => $easter,
+            'A doua zi de Paște' => $easter->addDay(),
+            'Paștele Blajinilor' => $easter->addDays(8),
+        ];
+    }
+    function orthodoxEaster(int $year): CarbonImmutable
+    {
+        $a = $year % 4;
+        $b = $year % 7;
+        $c = $year % 19;
+        $d = (19 * $c + 15) % 30;
+        $e = (2 * $a + 4 * $b - $d + 34) % 7;
+        $month = intval(floor(($d + $e + 114) / 31));
+        $day = (($d + $e + 114) % 31) + 1;
+
+        $gregorianEaster = CarbonImmutable::createFromDate($year, $month, $day);
```

Comment:
First the orthodox calender actually uses the julian calender not the greogrian hence your variable is named wrongly. However if you want to get orthodox eastern you can use php's build in predefined constants:

```php
return CarbonImmutable::createFromTimestamp(easter_date($year, CAL_EASTER_ALWAYS_JULIAN))
```

Reply:
you are correct i am sorry about the confusion